flytekit/core/python_auto_container.py

```diff
@@ -141,24 +157,6 @@ def get_container(self, settings: SerializationSettings) -> _task_model.Containe
         )
 
 
-def _default_command(task: PythonAutoContainerTask, settings: SerializationSettings) -> List[str]:
```

Comment:
why did you move this back into the class?

Reply:
so I can call <task>.get_default_command() for fast serialize. seemed more oop-y this way